sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -232,11 +233,12 @@ def submit_helper(self, call, logpath, rqmt, name, task_name, start_id, end_id,
         :param int step_size:
         """
         name = self.process_task_name(name)
-        sbatch_call = ["sbatch", "-J", name, "-o", logpath + "/%x.%A.%a", "--mail-type=None"]
+        out_log_file = logpath + "/%x.%A.%t.%a"
```

Comment:
this will break all running jobs when sisyphus is updated?

Reply:
Ah, yeah. Probably need a fallback here as well.

Reply:
I think I would only include this `%t` if there are multiple tasks.